apps/pwabuilder/src/script/components/windows-form.ts

```diff
@@ -274,6 +299,78 @@ export class WindowsForm extends AppPackageFormBase {
     }
   }
 
+  async updateActionsSelection(checked: boolean) {
+    this.showUploadActionsFile = checked;
+
+    if (!checked) {
+      delete this.packageOptions.webActionManifestFile;
+      actionsSchemaValidation = null;
+    } else {
+      try {
+        const SCHEMA_ID = "https://aka.ms/appactions.schema.json";
+        const SCHEMA_URL = "https://raw.githubusercontent.com/microsoft/App-Actions-On-Windows-Samples/refs/heads/main/schema/ActionsSchema.json";
```

Comment:
Should this URL resolve?

Reply:
Waiting on green light from Actions team. Should be public once they push.